### PR TITLE
Kdeconnect events

### PR DESCRIPTION
**Pull request by valdur55**
Increase module responsiveness by using dbus signals to update module when needed. 
Reduce unnecessary dbus calls by looking what parts are used in format.

Yes, i could use only one function to trigger self.py3.update() but in one day I will use them with right purpose.

@florencebiree can you test it?

### CODE REVIEW

**Comment by florencebiree** [comment]
Hi, I just tried it, now net_type is "Unknown"… (4G in the phone screen). I'm using kdeconnect 22.12.0-2 on Debian sid.

**Comment by florencebiree** [comment]
Now it works well (nothing changed on my computer)… maybe some kdeconnect glitch…

File: py3status/modules/kdeconnector_async.py [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+        self._result = {}
+
+        self._signal_reachable_changed = None
+        self._signal_battery = None
+        self._signal_notifications = None
+        self._signal_conn_report = None
+
+        self._format_contains_notifications = self.py3.format_contains(
+            self.format, "notif_size"
+        ) or self.py3.format_contains(self.format, "notif_status")
```
Comment (lasers) [inline py3status/modules/kdeconnector_async.py:129]:
```suggestion
        self._format_contains_notifications = self.py3.format_contains(
            self.format, ["notif_size", "notif_status"]
        )
```

File: py3status/modules/kdeconnector_async.py [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
+
+        self._format_contains_connection_status = self.py3.format_contains(
+            self.format, "net_type"
+        ) or self.py3.format_contains(self.format, "net_strength")
```
Comment (lasers) [inline py3status/modules/kdeconnector_async.py:133]:
```suggestion
        self._format_contains_connection_status = self.py3.format_contains(
            self.format, ["net_strength", "net_type"]
        )
```

**Comment by valdur55** [comment]
Hey community, I think code is now ready for production.
@florencebiree can you test lastest version?

**Comment by florencebiree** [comment]
It works like a charm, thanks!

**Comment by ultrabug** [comment]
Thanks for yet another great work @valdur55 and thanks @florencebiree for testing

**Pull request merged**